### PR TITLE
add new example for InjectionParametersGenerator and its all-sky child

### PR DESCRIPTION
**Pull request by dbkeitel**
- demonstrate both classes
- draw once, make SFTs
- draw 1000 times, plot sky distribution

PyFstat_example_InjectionParametersGenerator_allsky

### CODE REVIEW

File: examples/other_examples/PyFstat_example_InjectionParametersGenerator.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+Ndraws = 1000
+phase_parameters = [phase_params_generator.draw() for n in range(Ndraws)]
+Alphas = np.array([p["Alpha"] for p in phase_parameters])
+Deltas = np.array([p["Delta"] for p in phase_parameters])
+plotfile = os.path.join(outdir, label + "_allsky.png")
+print(f"Plotting sky distribution of {Ndraws} points to file: {plotfile}")
+plt.subplot(111, projection="aitoff")
+plt.plot(Alphas - np.pi, Deltas, ".")
+plt.savefig(plotfile, dpi=300)
+plt.close()
```
Comment (Rodrigo-Tenorio) [inline examples/other_examples/PyFstat_example_InjectionParametersGenerator.py:72]: Very nice!

I'd add a histogram on `Alpha` and `np.sin(Delta)`.

Comment (dbkeitel) [reply examples/other_examples/PyFstat_example_InjectionParametersGenerator.py:72]: good idea

**Comment by dbkeitel** [comment]
PyFstat_example_InjectionParametersGenerator_alpha_hist
PyFstat_example_InjectionParametersGenerator_delta_hist
PyFstat_example_InjectionParametersGenerator_sindelta_hist

**Comment by dbkeitel** [comment]
With 10k points and extra histograms:

PyFstat_example_InjectionParametersGenerator_allsky
PyFstat_example_InjectionParametersGenerator_alpha_hist
PyFstat_example_InjectionParametersGenerator_delta_hist
PyFstat_example_InjectionParametersGenerator_sindelta_hist

**Review by Rodrigo-Tenorio — APPROVED** [review]
I'm usually quite opinionated about going for OOP matplotlib rather than this pylab style, but that wouldn't simplify a thing so no problems.

**Pull request merged**